spec/examples.txt

```diff
@@ -1,5 +1,7 @@
-example_id                                                 | status | run_time        |
----------------------------------------------------------- | ------ | --------------- |
-./spec/controllers/home_controller_spec.rb[1:1:1]          | passed | 0.00724 seconds |
-./spec/controllers/home_controller_spec.rb[1:1:2]          | passed | 0.00704 seconds |
-./spec/controllers/registrations_controller_spec.rb[1:1:1] | passed | 0.20941 seconds |
+example_id                                                 | status  | run_time        |
```

Comment:
# Ignore all logfiles and tempfiles.
/log/*
/tmp/*
!/log/.keep
!/tmp/.keep
/spec/examples.txt


Whats wrong with this?  Do I not need the / in the beginning?